Source/Core/RequestScheduler.js

```diff
@@ -217,7 +218,12 @@ function cancelRequest(request) {
   var active = request.state === RequestState.ACTIVE;
   request.state = RequestState.CANCELLED;
   ++statistics.numberOfCancelledRequests;
-  request.deferred.reject();
+  // check that deferred has not been cleared since cancelRequest can be called
+  // on a finished request, e.g. by clearForSpecs during tests
+  if (defined(request.deferred)) {
+    request.deferred.reject();
+    request.deferred = undefined;
```

Comment:
Can you add a comment here so that another developer doesn't come along and "streamline" the code by removing "unnecessary" undefined assignments.  Thanks!  Something simple like:

> explicit undefined assignment to ensure GC. see: https://github.com/CesiumGS/cesium/issues/8843